.github/workflows/pr-cleanup.yaml

```diff
@@ -0,0 +1,27 @@
+name: Unpublish Pull Request NPM tag
+
+on:
+  pull_request_target:
+    types:
+      - closed
+
+jobs:
+  unpublish-pr:
+    name: "Unpublish Pull Request NPM tag"
+    runs-on: ubuntu-latest
+    permissions: write-all
+    steps:
+      - name: CI Setup
+        uses: ./.github/actions/ci-setup
+
+      - name: Ensure NPM access
+        run: npm whoami
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Unpublish NPM tag: pr-${{ github.event.pull_request.number }}
+        id: unpublish
+        run: npm --force unpublish pr-${{ github.event.pull_request.number }}
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
      - name: Get package versions for PR tag
        id: pr_versions
        run: |
          VERSIONS=$(npm view fuels --json)
          PR_VERSIONS=$(echo "$VERSIONS" | tr ',' '\n' | tr -d '[]"' | grep -F "pr-${{ github.event.pull_request.number }}" | xargs)
          echo "::set-output name=versions::$PR_VERSIONS"
          
      - name: Unpublish NPM PR Tag Versions
        if: steps.pr_versions.outputs.versions != ''
        run: |
          for VERSION in ${{ steps.pr_versions.outputs.versions }}
          do
            echo "Unpublishing version $VERSION"
            npm --force unpublish fuels@${VERSION}
          done
        env:
          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
A quick note about the `::set-output` command:

> **Warning**: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

The warning above comes from [here](https://github.com/FuelLabs/fuels-ts/actions/runs/4779747120/jobs/8497023538?pr=827#step:6:21).

> **Note** The [`pr-release.yaml`](https://github.com/FuelLabs/fuels-ts/blob/master/.github/workflows/pr-release.yaml) workflow also uses this command.

Reply:
Follow-up issue for existent workflows:
 - https://github.com/FuelLabs/fuels-ts/issues/928